datasets/GM1896_zwartewaterland/GM1896_zwartewaterland.derived.ad

```diff
@@ -28,6 +28,9 @@
 - has_aggregated_chemical_industry = false
 - has_detailed_chemical_industry = true
 - has_coal_oil_for_heating_built_environment = false
+- has_aquathermal_potential_for_surface_water = false
```

Comment:
These area attributes aren't necessary on the `master` branch, are they? But I guess it won't hurt to keep them here for when we're merging the `aquathermal` branch into `master`.